plugins/bounce.js

```diff
@@ -116,8 +116,8 @@ exports.single_recipient = function (next, connection) {
         return next();
     }
 
-    connection.loginfo(plugin, "bounce with too many recipients to: " +
-        connection.transaction.rcpt_to.join(','));
+    connection.loginfo(plugin, `bounce with too many recipients to: 
+        ${connection.transaction.rcpt_to.join(',')}`);
```

Comment:
This change introduces an extra \n in the string.

Reply:
What would be the preferred way of resolving this?
(See my new commit)